src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests/SecureParamsInNestedDeploymentsTests.cs

```diff
@@ -0,0 +1,749 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Analyzers.Linter.Rules;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bicep.Core.UnitTests.Diagnostics.LinterRuleTests
+{
+    [TestClass]
+    public class SecureParamsInNestedDeploymentsTests : LinterRuleTestsBase
```

Comment:
Rename to SecureParamsInNestedDeploymentsRuleTests to keep it consistent with other tests?

Reply:
done